src/View/Helper/FormHelper.php

```diff
@@ -386,6 +386,10 @@ public function create(mixed $context = null, array $options = []): string
             unset($options['context']);
         }
 
+        if (isset($options['method'])) {
```

Comment:
Should `method` overwrite type if both are provided? I would expect that `method` only overwrites `type` if `type` is undefined.

Reply:
If I overwrite `type` with `method` only if the former is undefined then the test case which I fix still ends up with `'method' => 'put'`.

Reply:
Error sorry, that test case ends up with `'method' => 'get'`. 

Though the description of the test method `testCreateExplicitMethodEnctype` says `Explicit method overwrites inferred method from 'type'` and that's what my patch does.

IMO `method` should override `type` if both are set.

Reply:
My bad, I should have figured that out. Makes sense.